packages/metadata-fields/FeaturedMetadata.jsx

```diff
@@ -253,10 +253,7 @@ export default function FeaturedMetadata({ data }) {
         </div>
 
         {firstRow.length === 0 ? (
-          <hr
-            className="su-bord
```

Comment:
Looks like a line break was accidentally introduced in `develop` so I'm fixing it here